README.md

```diff
@@ -151,6 +151,29 @@ A wide range of lockfiles are supported by utilizing this [lockfile package](htt
 $ osv-scanner --lockfile=/path/to/your/package-lock.json --lockfile=/path/to/another/Cargo.lock
 ```
 
+The scanner will attempt to automatically determine the parser to use for each 
+file based on the filename (except for `installed`, see below); you can explicitly
+tell the scanner what parser to use for specific files by prefixing the path with 
+the name of the file to parse it as followed by a colon:
+
+```bash
+$ osv-scanner --lockfile 'requirements.txt:/path/to/your/extra-requirements.txt'
+```
+
+You can use this to scan the lockfile used by the Alpine Package Keeper (apk):
+
+```bash
+$ osv-scanner --lockfile 'installed:/lib/apk/db/installed'
```

Comment:
I wonder if we should rename "installed" to something less generic. This might come back to bite us in the future..

@another-rex @G-Rath thoughts?

Reply:
```
alpine-installed
```
maybe?

Reply:
Or perhaps just "apk-installed", if this isn't really specific to Alpine distro but rather the APK packaging mechanism instead, which might be re-used by other distros?

Reply:
for now I'll go with `apk-installed` because that matches the name of the actual Go file